scripts/remove/authelia.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# |& was added to Bash 4 as an abbreviation for 2>&1 |. - https://tldp.org/LDP/abs/html/io-redirection.html
+#
+if [[ ! -f /install/.authelia.lock ]]; then
+    echo_warn "Authelia is not installed"
+    exit 1
+else
+    systemctl disable -q --now authelia |& tee -a "${log}" &> /dev/null
+    rm -rf /etc/authelia
```

Comment:
don't forget to remove the symlink in /etc/systemd/system as that will remain there and be all broken and sad